content/pt/docs/concepts/cluster-administration/cluster-administration-overview.md

```diff
@@ -0,0 +1,80 @@
+---
+reviewers:
+- davidopp
+- lavalamp
+title: Visão Geral da Administração de Cluster
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+A visão geral da administração de cluster é para qualquer um criando ou administrando um cluster Kubernetes.  Assume alguma familiaridade com os [conceitos](/docs/concepts/) centrais do Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+## Planejando um cluster
+
+Veja os guias em [Setup] (/docs/setup/) para exemplos de como planejar, configurar e configurar  
```

Comment:
I believe you need remove the space here `[Setup] (/docs/setup/)`

Reply:
Diego, please, remove the duplicated word "configurar".

Good job!

Reply:
@femrtnz @jesus-amido Thanks for reviewing.
For this sentence I feel that I missing something. I put the two 'configurar' on purpose so we can discuss the meaning of 'set up'.  The original says:  
"See the guides in [Setup](/docs/setup/) for examples of how to plan, __set up__, and configure Kubernetes clusters."

Do you agree if I say: "...para exemplos de como planejar, __ativar__ e configurar ..." 

Can you come up with another verb for __set up__?

Reply:
Looking at Google Translator result, I think we can use "**iniciar**" for set up.

See this link with search at GT.
[translate result](https://translate.google.com.br/#view=saved&op=translate&sl=en&tl=pt&text=set%20up)

Reply:
I would go with "Olha o guia de Setup para exemplos de como planejar, configurar ...."